Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.13
+
+EXPOSE 9135
+
+WORKDIR /go/src/app
+COPY . .
+
+RUN go get -d -v ./...
```

Comment:
I am pretty sure go install takes care of this. Also, shouldn't GO111MODULE=on be set?

Reply:
I don't know enough about go mod. I saw it being set in https://dev.to/plutov/docker-and-go-modules-3kkn, so I added it here too.